src/amo/sagas/reviews.js

```diff
@@ -51,6 +59,37 @@ function* fetchReviews({
   }
 }
 
+function* fetchUserReviews({
+  payload: { errorHandlerId, page, userId },
+}: FetchUserReviewsAction): Generator<any, any, any> {
+  const errorHandler = createErrorHandler(errorHandlerId);
+
+  try {
+    const state = yield select(getState);
+
+    const params: GetReviewsParams = {
+      apiState: state.api,
+      // Hide star-only ratings (reviews that do not have a body).
```

Comment:
Hmm, maybe we should show star-only ratings for user reviews? I'm not sure. It could help someone if they were trying to edit a rating. In any case, it's a future considering for UX I suppose.

Reply:
Good question.. I'll ask @jvillalobos